### ISSUE TITLE
Delay JavaScript Execution compatibility with the Essential Add-on registration form

### ISSUE THREAD

**Issue by DahmaniAdame** [wp-media/wp-rocket#4699]
**Before submitting an issue please check that you’ve completed the following steps:**
- [x] Made sure you’re on the latest version `3.10.7`
- [x] Used the search feature to ensure that the bug hasn’t been reported before

**Describe the bug**
When the [Essential Add-on](https://essential-addons.com/elementor/login-register-form) registration form is used as a modal, half of the form is cut when Delay JavaScript Execution is enabled with [Elementor's exclusions](https://docs.wp-rocket.me/article/1560-delay-javascript-execution-compatibility-exclusions#elementor-pro).

![image](https://user-images.githubusercontent.com/1289174/151970343-b4ded4e2-019a-40c4-bb0c-b06cecdee2e4.png)

Instead of: 

![image](https://user-images.githubusercontent.com/1289174/151970360-67f661c1-f80f-4754-91f1-2b4ddea88310.png)

Excluding the following from Delay JavaScript Execution fixes the issue: 

```
/uploads/essential-addons-elementor/(.*).min.js
```

**To Reproduce**
1. Add the Essential add-on login form as a modal
2. Add Elementor's exclusions
3. Click the bottom to trigger the modal

**Expected behavior**
The modal should work properly when Delay JavaScript Execution is used with Elementor's exclusions.

**Screenshots**
Added to the issue description.

**Additional context**
Ticket - https://secure.helpscout.net/conversation/1771587879/322641?folderId=3864735

**Backlog Grooming (for WP Media dev team use only)**
- [ ] Reproduce the problem
- [ ] Identify the root cause
- [ ] Scope a solution
- [ ] Estimate the effort

**Comment by piotrbak** [comment]
@DahmaniAdame Is there any way to have a more specific exclusion?

I'm thinking that some files from `/essential-addons-elementor/(.*).js` might be jQuery dependent. It could cause some issues in other scenarios

**Comment by DahmaniAdame** [comment]
@piotrbak it's actually one file but since it's dynamic, we need to exclude the above pattern. 
Example: 
```
/wp-content/uploads/essential-addons-elementor/cb70d11b8.min.js
```

**Comment by DahmaniAdame** [comment]
Updated to `/uploads/essential-addons-elementor/(.*).min.js` to be more accurate.

**Comment by piotrbak** [comment]
Will be fixed with the 3.13 backend

**Comment by piotrbak** [comment]
Handled in 3.13 using dynamic lists
